src/renderer/views/Watch/Watch.js

```diff
@@ -988,6 +990,18 @@ export default Vue.extend({
       return Math.floor(this.getWatchedProgress())
     },
 
+    updateTitle: function () {
+      document.title = `${this.videoTitle} - FreeTube`
+
+      // simulates a reload to update the title after fetching it
+      this.$router.go({
```

Comment:
Is there a reason this is needed?  Just changing the title without this router chunk works fine for me.

Reply:
I added that when I first couldn't get it working. That was when I was calling updateTitle in the wrong location. You are correct, that portion is not needed. I have amended the code to get rid of it.